internal/page/donor/payment_confirmation.go

```diff
@@ -58,9 +58,14 @@ func PaymentConfirmation(logger Logger, tmpl template.Template, payClient PayCli
 		} else {
 			lpa.Tasks.PayForLpa = actor.PaymentTaskPending
 
-			for i, evidence := range lpa.Evidence.Documents {
-				if evidence.Sent.IsZero() {
-					err := evidenceS3Client.PutObjectTagging(r.Context(), evidence.Key, []types.Tag{
+			documents, err := documentStore.GetAll(r.Context())
```

Comment:
do we need all of this since it happens in `payHelper.Pay`?

Reply:
It does, but not for `HalfFee` - we wanted to be sure that the payment had been successful before tagging the docs as being ready for replication hence the replication.